__tests__/lint/assertions-casing-snake-case-error/.redocly.yaml

```
@@ -4,16 +4,17 @@ apis:
 
 lint:
   rules:
-    assertions:
-      - context:
-          - type: Operation
-            matchParentKeys: [get]
-        subject: Operation
-        property: operationId
-        message: Operation id should be snake_case
-        casing: PascalCase
-      - subject: Operation
-        property: operationId
-        message: Operation id should be snake_case
-        casing: snake_case
+    assert/operation-id-pascal-case:
+      context:
+        - type: Operation
+          matchParentKeys: [get]
+      subject: Operation
+      property: operationId
+      message: Operation id should be PascalCase
```

This one seems like the message should be improved to help document the functionality better. Corresponding snapshots should be updated too.

```suggestion
      message: Operation id for get requests should be PascalCase
```